tests/end-to-end/api/01-users.js

```diff
@@ -20,6 +20,13 @@ import { updatePermission, updateSetting } from '../../data/permissions.helper';
 import { createUser, login, deleteUser, getUserStatus } from '../../data/users.helper.js';
 import { createRoom } from '../../data/rooms.helper';
 
+const reservedWords = [
```

Comment:
can you move this array to `tests/data/api-data.js` ? this way you can use it here and on `tests/end-to-end/api/02-channels.js` without having to duplicate it =)